src/Components/Header.tsx

```diff
@@ -214,8 +214,38 @@ const Header = () => {
             </Link>
             {currentUser ? (
               <>
-                <Box onClick={handleSubmit}>
-                  <Typography sx={itemsDesk}>Logout</Typography>
+                <Box sx={{ display: "flex", alignItems: "center" }}>
+                  <Link to="/profile/:id" style={{ textDecoration: "none" }}>
+                    {userImg ? (
+                      <Box
+                        component="img"
+                        src={userImg}
+                        alt={userName?.charAt(0)}
```

Comment:
Where does userName come from?
And why do you only want to display the first letter?